ipaserver/install/ipa_migrate_constants.py

```diff
@@ -0,0 +1,1034 @@
+        'oc': ['ipasudorule'],
+        'subtree': ',cn=sudorules,cn=sudo,$SUFFIX',
+        'label': 'Sudo Rules',
+        'mode': 'all',
+        'count': 0,
+    },
+    'sudo_cmds': {
+        'oc': ['ipasudocmd'],
+        'subtree': ',cn=sudocmds,cn=sudo,$SUFFIX',
+        'label': 'Sudo Commands',
+        'mode': 'all',
+        'count': 0,
+    },
+    'sudo_cmd_groups': {
+        'oc': ['ipasudocmdgrp'],
+        'subtree': ',cn=sudocmdgroups,cn=sudo,$SUFFIX',
+        'label': 'Sudo Command Groups',
+        'mode': 'all',
+        'count': 0,
+    },
+    # DNS
+    'dns_container': {
+        'oc': ['ipadnscontainer', 'nscontainer'],
+        'subtree': 'cn=dns,$SUFFIX',
+        'label': 'DNS Container',
+        'mode': 'all',
+        'count': 0,
+    },
+    'dns_server_container': {
+        'oc': ['nscontainer'],
+        'subtree': 'cn=servers,cn=dns,$SUFFIX',
+        'label': 'DNS Server Container',
+        'mode': 'all',
+        'count': 0,
+    },
+    'dns_records': {
+        'oc': ['idnsrecord', 'idnszone'],
+        'subtree': ',cn=dns,$SUFFIX',
+        'label': 'DNS Records',
+        'mode': 'all',
+        'count': 0,
+    },
+    'dns_servers': {
+        'oc': ['idnsServerConfigObject'],
+        'subtree': ',cn=servers,cn=dns,$SUFFIX',
+        'label': 'DNS Servers',
+        'mode': 'all',
+        'count': 0,
+    },
+    # Kerberos
+    'krb_realm': {
+        'oc': ['krbrealmcontainer'],
+        'subtree': ',cn=kerberos,$SUFFIX',
+        'label': 'Kerberos Realm',
+        'mode': 'all',
+        'count': 0,
+    },
+    'kerberos_policy': {  # principal
+        'oc': ['krbticketpolicyaux'],
+        'subtree': ',cn=kerberos,$SUFFIX',
+        'label': 'Kerberos Policy',
+        'mode': 'all',
+        'count': 0,
+    },
+    'krb_pwpolicy': {
+        'oc': ['ipapwdpolicy'],
+        'subtree': 'cn=global_policy,cn=$REALM,cn=kerberos,$SUFFIX',
+        'label': 'Kerberos Password Policy',
+        'mode': 'all',
+        'count': 0,
+    },
+    'krb_default_pwppolicy': {
+        'oc': ['krbpwdpolicy'],
+        'subtree': 'cn=default kerberos service password policy,'
+                   'cn=$REALM,cn=kerberos,$SUFFIX',
+        'label': 'Kerberos Default Password Policy',
+        'mode': 'all',
+        'count': 0,
+    },
+
+    # Other config
+    'master_configs': {
+        'oc': ['ipaconfigobject'],
+        'subtree': ',cn=masters,cn=ipa,cn=etc,$SUFFIX',
+        'label': 'Master Configurations',
+        'mode': 'all',
+        'count': 0,
+    },
+
+    'domain_config': {
+        'oc': ['ipadomainlevelconfig'],
+        'subtree': 'cn=domain level,cn=ipa,cn=etc,$SUFFIX',
+        'label': 'Domain Configuration',
+        'mode': 'all',
+        'count': 0,
+    },
+
+    # Profiles
+    'profiles': {
+        'oc': ['duaconfigprofile'],
+        'subtree': ',ou=profile,$SUFFIX',
+        'label': 'DUA Config Profiles',
+        'mode': 'all',
+        'count': 0,
+    },
```

Comment:
I think migrating this profiles is fine but I don't think we need to report on it. This is there for HP/ux and Solaris support which at this point is a bit moot. So fine in xfering it but I don't think we should report it. I imagine it would cause confusion.

Reply:
More thinking on this. I think the summary is very nice but I imagine that some people, I did, are going to want to be able to drill down into what these mean. E.g. I had 1 PBAC privilege migrate but nothing else PBAC. My remote instance has been around longer than user so maybe I added a privilege, but why no permissions too? It's there in ipa-migrate.log but you have to know the DIT well enough to translate the text "PBAC privilege" properly. In this case it is DNS-enabled remote and non-DNS local.

Reply:
> I think migrating this profiles is fine but I don't think we need to report on it. This is there for HP/ux and Solaris support which at this point is a bit moot. So fine in xfering it but I don't think we should report it. I imagine it would cause confusion.

It's not designed to work that way and would require some hacking to make that work.  I prefer to remove it from object list.  The point of migration to get to a new system, so we should not be dealing with Solaris/HP at this stage.

Reply:
HBAC is in there.  Not sure what you mean.

But honestly I don't like migrating something and not reporting that it happened.  If items are being migrated that shouldn't please let me know which ones exactly.